www/src/components/showcase-details.js

```diff
@@ -470,6 +474,7 @@ const ShowcaseDetails = ({ parent, data, isModal, categories }) => (
                         verticalAlign: `sub`,
                       }}
                     />
+                    {` `}
                     Visit site
                     {` `}
```

Comment:
Why put spaces around " Visit site "?

Reply:
The one on the left, was there, so I didn't feel like removing, the one before is to give icon some breathing.